packages/bigtable/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@google-cloud/bigtable",
-  "version": "0.8.0",
+  "version": "0.9.0",
```

Comment:
```
cd packages/{module}
npm run publish-module 0.9.0
```

Some day, I'd like to have it support the raw arguments to `npm version` (which is `major`, `minor`, or `patch`), but for now we only accept the new version. The script also runs a quick sanity check to confirm the new version will be *newer* than the last published version.